README.md

```diff
@@ -24,6 +26,22 @@ Add following step to your workflow:
 
 Specifies number of latest releases (sorted by `created_at`) to keep. Pass `0` if you want to delete all releases
 
+#### `keep_min_download_counts`
+
+| required | default |
+| -------- | ------- |
+| false    |    0    |
+
+Specifies that versions with fewer downloads than this value (sorted by "created_at") are to be removed. If you do not need this restriction, please pass "0" or do not set the variable. The rule is less than, not less than or equal to, and values greater than or equal to will be retained.
```

Comment:
Let's word this more simply.  Something like "Specifies the number of downloads that versions must have at minimum, otherwise they will be removed ...."

Reply:
The function of "keep latest" is to keep the latest releases, while "delete_expired_data" is to delete releases older than how many days. These two are different concepts.
The reason why "delete_expired_data" is added is because there is a limit of `keep_min_download_counts`. For example, there are 100 releases in a certain repository, and "keep latest" is 10, then the last 90 should be deleted, but if there are still `keep_min_download_counts`, and If the download volume of the last 90 releases is greater than 5, then these 100 releases will not be deleted. If delete_expired_data is added, the setting value is 10. If the creation time of the last 90 releases is greater than 10 days compared to the current time, Then the next 90 releases will be deleted regardless of whether the download volume is greater than 5.